_schemaV2.graphql

```diff
@@ -16314,6 +16315,21 @@ type UserEdge {
   node: User
 }
 
+type UserFollows {
+  artistFollowsConnection(
```

Comment:
this might just be called `artistsConnection` and `genesConnection` (plural `artist` and `gene` and no need for the redundant 'follows' in the name as it's part of the `UserFollows` type)